src/views/Home.vue

```diff
@@ -1,5 +1,34 @@
 <template>
-  <div class="container">
+  <div class="container gt-sm">
+    <q-splitter v-model="splitterModel" disable style="height: 85vh" separator-style="visibility: hidden">
+      <template #before>
+        <div class="no-scroll no-scroll" style="overflow: hidden">
+          <img alt="Main Photo" src="@/assets/duck.jpg" class="main-photo" />
+        </div>
+      </template>
+      <template #after>
+        <div class="q-pa-md">
+          <header-typography msg="J & P" />
+          <q-separator color="black" inset />
+          <header-typography id="details" msg="Wedding Details" />
+          <card-items :card-infos="cardInfos" />
+          <header-typography id="story" msg="Our Story" />
+          <div style="padding: 5em">Hello</div>
+          <header-typography id="schedule" msg="Schedule" />
+          <div style="padding: 5em">Hello</div>
+          <q-separator color="black" inset />
+          <header-typography id="travel" msg="Travel Information" />
+          <div style="padding: 5em">O'HAre plane or amtrak water powered steam train</div>
+          <q-separator color="black" inset />
+          <header-typography id="accomodations" msg="Accomodations" />
+          <div style="padding: 5em">Hello</div>
+          <q-separator color="black" inset />
+          <header-typography id="gallery" msg="Gallery" />
+        </div>
+      </template>
+    </q-splitter>
+  </div>
+  <div class="container lt-md">
```

Comment:
I believe container has padding which is why there's a border. I don't think container is the best class to use here since it's also used for card type I think? At least I'm using it for password container

Reply:
Ooooooooh, let me try to fix this.

Reply:
Sadly the border is still there, I think there's some hidden class somewhere doing something lol. I'll write up a task for it.